### DISCUSSION TITLE
Need help on a project

### DISCUSSION THREAD

**Discussion by Hassan78666** [apache/netbeans#6101] [Q&A]
Hello i am working on a 2d game project called cookgame and my project wont open up and its soo annoying and it keeps on saying this Error: Could not find or load main class Customer.java.java
Caused by: java.lang.ClassNotFoundException: Customer.java.java 

How shall i encounter this issue

**Comment by asbachb** [comment]
I guess you need to provide some more information. For me it's unclear:
* What kind of project is it? Ant, Maven?
* How did you create the project? (As you wrote you want to open the project)
* Can you upload the entire project folder?
* Can you provide screenshots of the problem?

The doubled `java` extension looks weird.

**Reply by Hassan78666** [reply]
Yeah so its an 2D Game Project and the codes was created by our tutor

**Reply by Hassan78666** [reply]
We Just had to develop them task by task

**Reply by Hassan78666** [reply]
and yes i can upload the entire project and show you what the problem is as im struggling for dear Life

**Reply by asbachb** [reply]
How you try to open it? I just see a folder with a set of (wrongly) named java class files. That `java.java` in the end looks quite suspicious though.

**Reply by Hassan78666** [reply]
Its really weird because normally i try to save my files all of them in .java and etc but however i have tried my very best to get to task 8 and happy with my progress a lot of progress has been made and its not been very easy to be honest with you.

**Reply by Hassan78666** [reply]
because i had to keep on deleting the old files and making new ones to be honest

**Reply by Hassan78666** [reply]
and i kept getting a lot of exceptions and got rid of them and now i have this issue now warning: [options] bootstrap class path not set in conjunction with -source 8
1 warning 

and i dont know how to fix it any ideas to this solution

**Comment by Hassan78666** [comment]
[Fopcoursework 1 Hmehmood back up.zip](https://github.com/apache/netbeans/files/11832122/Fopcoursework.1.Hmehmood.back.up.zip)

**Comment by Hassan78666** [comment]
I would have sent you the screenshots but netbeans doesnt look like it can regognize the files sorry.

**Comment by Hassan78666** [comment]
Yes I know God knows why but I re-done the project again from scratch and made a lot of progress to it.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by emaphis** [comment]
How have you been working on projects in class?  NetBeans needs you're java files in some form of project.  NetBeans defaults to Maven projects. 
Create a CookGame Maven project.
Create a "uk.ac.bradford.cookgame" package.
Move your files to the "uk.ac.bradford.cookgame" package on your hard drive.
Get rid of the extra x.java extension and go from there.

**Reply by Hassan78666** [reply]
Thank you for your help and it worked out out

**Reply by Hassan78666** [reply]
If I need help in the future I will Let you guys know
